tests/unit/fem/test_assembly_levels.cpp

```diff
@@ -111,13 +111,13 @@ TEST_CASE("Assembly Levels", "[AssemblyLevel], [PartialAssembly]")
 {
    auto assembly = GENERATE(AssemblyLevel::PARTIAL, AssemblyLevel::ELEMENT,
                             AssemblyLevel::FULL);
-   auto pb = GENERATE(0, 1, 2);
-   auto dg = GENERATE(true, false);
-   auto order_2d = GENERATE(2, 3, 4);
+   auto order_2d = GENERATE(2, 3);
    auto order_3d = GENERATE(2);
 
    SECTION("2D")
    {
+      auto pb = GENERATE(0, 1, 2);
```

Comment:
It looks like `pb` and `dg` are now in the 2D and 3D sections with the same values as before. Did you intend for them to be different in these two cases?

Reply:
So `GENERATE` makes nested loops, even if the variable is not used for a test? Does that mean `order_2d` and `order_3d` are also nested loops, which would make the 3D case test run twice (since `order_2d` has size 2)?

Reply:
Good catch! I didn't notice that, yes that is exactly what that means.

I fixed it now, runtime is even shorter.

Reply:
Though, we may want to test dg now, as the PR has just been merged for AMR.